src/Datadog.Trace/TraceContext.cs

```diff
@@ -150,6 +150,7 @@ private void DecorateRootSpan(Span span)
                 span.SetTag(Tags.AzureAppServicesInstanceName, AzureAppServices.Metadata.InstanceName);
                 span.SetTag(Tags.AzureAppServicesOperatingSystem, AzureAppServices.Metadata.OperatingSystem);
                 span.SetTag(Tags.AzureAppServicesRuntime, AzureAppServices.Metadata.Runtime);
+                span.SetTag(Tags.AzureAppServicesExtensionVersion, AzureAppServices.Metadata.SiteExtensionVersion);
```

Comment:
Can this be `null` (e.g. if it's an older version of the extension installed)? If so, is there any value in recording that as a key without a value?

Reply:
Probably if this is null we are in a situation where there is a newer nuget and an older extension and the traces are detached.
I find value in knowing that we are on a tracer version which tries to set these items but fails. 
It differentiates between old versions and new versions with a problem.

If we decide to add the tracer version to the tags as well, that would be brilliant IMO.